bin/test_python.py

```diff
@@ -1,3 +1,23 @@
+import os
+TEST_SYMPY = os.getenv('TEST_SYMPY', False)
+
 import symengine
 if not symengine.test():
     raise Exception('Tests failed')
+
+try:
```

Comment:
Can you remove this try statement and move the lines below to under `if TEST_SYMPY`?